pkg/podexecutor/staticpod.go

```diff
@@ -284,4 +295,45 @@ func chownr(path string, uid, gid int) error {
 
 func etcdNameFile(dataDir string) string {
 	return filepath.Join(dataDir, "server", "db", "etcd", "name")
-}
\ No newline at end of file
+}
+
+func writeDefaultPolicyFile(policyFilePath string) error {
+	auditPolicy := auditv1.Policy{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Policy",
+			APIVersion: "audit.k8s.io/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{},
+		Rules: []auditv1.PolicyRule{
```

Comment:
I don't think we should configure any rules by default, should we? CIS just says you have to enable logging and configure a policy, it doesn't say that the policy needs to have any rules. If it **does** need to have rules, can we add a comment with a reference to what these are? Auditing just pods by default seems awfully arbitrary.

Reply:
kubeapi doesnt accept a policy with no rule, it fatals with an error that you should add at least one rule